packages/sdk/src/swap/assets.ts

```diff
@@ -24,10 +23,8 @@ export const eth$: AssetFn = (network, env) => ({
   symbol: 'ETH',
   chainflipId: 'ETH',
   isMainnet: !isTestnet(network),
-  minimumSwapAmount: readMinimumSwapAmount(env, {
-    chain: 'Ethereum',
-    asset: 'ETH',
-  }).toString(),
+  minimumSwapAmount:
```

Comment:
should we rename this property of the `TokenData` to `minimumDepositAmount`?

Reply:
No, for this context it is the minimum swap amount